.vscode/launch.json

```diff
@@ -1,36 +1,53 @@
 {
   "version": "0.2.0",
   "configurations": [
+    
+    
+      //TODO add tests debug configuration
+      //"program": "${cmake.testProgram}",
+      //"args": ["${cmake.testArgs}"],
+
     {
-      "name": "C/C++: Linux: build and debug FreeCAD",
+      "name": "C/C++ debugger",
       "type": "cppdbg",
       "request": "launch",
-      "program": "${workspaceFolder}/build/bin/FreeCAD",
-      "args": [],
-      "stopAtEntry": false,
       "cwd": "${workspaceFolder}",
-      "environment": [],
-      "externalConsole": false,
-      "preLaunchTask": "CMake: build"
-    },
-    {
-      "name": "C/C++: macOS: build and debug FreeCAD",
-      "type": "cppdbg",
-      "request": "launch",
-      "program": "${workspaceFolder}/build/bin/FreeCAD",
-      "args": [],
+      "program": "${command:cmake.launchTargetPath}", //This triggers a build as well
+      "args": ["${workspaceFolder}/.vscode/VSCodeAutostartDebug.FCMacro"], //if launching the Tests_run executable this parameter gets passed as well, but gtest does not seem to mind.
+      "environment": [
+        {
+          // add the directory where our target was built to the PATHs
+          // it gets resolved by CMake Tools:
+          "name": "PATH",
+          "value": "${env:PATH}:${command:cmake.getLaunchTargetDirectory}"
+        },
+        {
+          "name": "PYDEVD_DISABLE_FILE_VALIDATION",
+          "value": "1"
+        }
+      ],
+      "linux": {
+        "MIMode": "gdb",
+        "miDebuggerPath": "/usr/bin/gdb"
+      },
+      "osx": {
+        "MIMode": "lldb"
+      },
+      "windows": {
+        "MIMode": "gdb",
+        "miDebuggerPath": "C:\\MinGw\\bin\\gdb.exe"
+      },
       "stopAtEntry": false,
-      "cwd": "${workspaceFolder}",
-      "environment": [],
       "externalConsole": false,
-      "MIMode": "lldb",
-      "preLaunchTask": "CMake: build"
+      "presentation": {
+        "hidden": true,
+      }
     },
     {
-      "name": "Python: Remote Attach",
+      "name": "Python debugger",
       "type": "python",
       "request": "attach",
-      "preLaunchTask": "conda: activate environment",
```

Comment:
I'm not convinced that we should remove this already without providing instructions on how to get the conda stuff working. In any case comment it out

Reply:
It's not removed, it is present (commented out) just below in the compound task